src/tests/commonTests.ts

```diff
@@ -3,7 +3,7 @@ import { JSX } from "../components";
 import { toHaveNoViolations } from "jest-axe";
 import axe from "axe-core";
 import { config } from "../../stencil.config";
-import { GlobalTestProps, waitForAnimationFrame } from "./utils";
+import { GlobalTestProps } from "./utils";
```

Comment:
I can submit a PR for this to split it out.

Reply:
Created https://github.com/Esri/calcite-components/pull/4484